chapters/operatorsandexpressions.tex

```diff
@@ -804,7 +804,7 @@ \subsubsection{spatialDistribution}\label{spatialdistribution}
 solver.
 \end{nonnormative}
 
-\lstinline!spatialDistribution! allows to approximate efficiently the solution of the infinite-dimensional problem
+\lstinline!spatialDistribution! lets one approximate efficiently the solution of the infinite-dimensional problem
```

Comment:
I suppose this one was forgotten in the last commit? Maybe this could work:
```suggestion
\lstinline!spatialDistribution! allows efficient approximate solution of the infinite-dimensional problem
```

Reply:
It was missed, but I don't think that is the best option. I changed it more completely.